doc/examples/rendering/traceSetsArnold.gfr

```diff
@@ -0,0 +1,348 @@
+__children["RenderSetup"]["ArnoldOptions"]["__uiPosition"].setValue( imath.V2f( 7.22612143, -88.5188065 ) )
+__children["RenderSetup"]["in"].setInput( __children["ShaderAssignment"]["out"] )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["in"], 'nodule:color', imath.Color3f( 0.240099996, 0.339399993, 0.485000014 ) )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["in"], 'nodule:type', 'GafferUI::StandardNodule' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["in"], 'description', 'The input scene' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["in"], 'plugValueWidget:type', '' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["in"], 'noduleLayout:spacing', 2.0 )
+__children["RenderSetup"]["BoxOut"]["in"].setInput( __children["RenderSetup"]["Outputs"]["out"] )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'nodule:color', imath.Color3f( 0.240099996, 0.339399993, 0.485000014 ) )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'labelPlugValueWidget:renameable', True )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'nodule:type', 'GafferUI::StandardNodule' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxOut"]["__out"], 'description', 'The processed output scene.' )
+__children["RenderSetup"]["BoxIn"]["__in"].setInput( __children["RenderSetup"]["in"] )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'nodule:color', imath.Color3f( 0.240099996, 0.339399993, 0.485000014 ) )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'labelPlugValueWidget:renameable', True )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'nodule:type', 'GafferUI::StandardNodule' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'description', 'The input scene' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'plugValueWidget:type', '' )
+Gaffer.Metadata.registerValue( __children["RenderSetup"]["BoxIn"]["__in"], 'noduleLayout:spacing', 2.0 )
+__children["RenderSetup"]["__uiPosition"].setValue( imath.V2f( 8.73167801, -126.091125 ) )
+__children["Backdrop"]["title"].setValue( 'Example: Trace Sets (Arnold)' )
+__children["Backdrop"]["description"].setValue( "In this graph we show how to make use of Arnold's trace sets.\n\nArnold allows both objects and rays to be tagged with one or more trace sets. This allows the scene to be compartmentalized, such that affected shading calculations simply dont 'see' objects in other trace sets.\n\nIn this example we will create a mirror that only reflects some of the objects in the scene.\n\n - An object's trace set membership is tagged using Gaffer sets.\n\n - Rays cast from a shader are tagged by inserting a aiTraceSet shader into the shading network before assignment." )
```

Comment:
### Backdrop text
~~In this graph we show how to make use of Arnold's trace sets.~~

Arnold allows both objects and rays to be tagged with one or more trace sets. This allows the scene to be compartmentalized, such that affected shading calculations simply ~~dont 'see'~~ **don't "see"** objects in other trace sets.

In this example ~~we will create~~ **, we show how to make use of Arnold's trace sets, by creating** a mirror that only reflects some of the objects in the scene.

 - ~~An object's trace set membership is tagged using Gaffer sets.~~ **First, we add the spheres to a special set.**
**[delete blank line]**
 - Rays cast from a shader are tagged by inserting ~~a~~ **an** aiTraceSet shader **(via a TraceSet node)** into the shading network before assignment.